AZ3166/src/cores/arduino/az_iot/c-utility/inc/azure_c_shared_utility/xlogging.h

```diff
@@ -44,6 +44,7 @@ typedef void(*LOGGER_LOG_GETLASTERROR)(const char* file, const char* func, int l
 
 #define LOG_NONE 0x00
 #define LOG_LINE 0x01
+#define __SUB_FILE_PATH__  (strstr(__FILE__, "az_iot") ? strstr(__FILE__, "az_iot") : __FILE__)

```

Comment:
I think we don't need the subfolder, just the file name
#define __FILENAME__ (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__)